sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -186,72 +185,36 @@ def s3_event_handler(event):
-        science_file = ScienceFilepathManager(filename)
-    except InvalidScienceFileError as e:
-        logger.error(str(e))
-        return http_response(status_code=400, body=str(e))
-
+    # Check if the file is a valid science file or not
+    # TODO: change these lines once filename validator
+    # is implemented on sds-data-access repo and released
+    science_file = ScienceFilepathManager(filename)
     # setup a dictionary of metadata parameters to unpack in the
-    # file catalog table
+    # file catalog table. Eg.
+    # {
+    #     "file_path": None,
+    #     "instrument": self.instrument,
+    #     "data_level": self.data_level,
+    #     "descriptor": self.descriptor,
+    #     "start_date": datetime.strptime(self.startdate, "%Y%m%d"),
+    #     "end_date": datetime.strptime(self.enddate, "%Y%m%d"),
+    #     "version": self.version,
+    #     "extension": self.extension,
+    #     "ingestion_date": date_object,
+    # }
     metadata_params = science_file.get_file_metadata_params()
     metadata_params["file_path"] = s3_filepath
 
-    if metadata_params["data_level"] != "l0":
-        logger.error("Invalid data level. Expected l0")
-        return http_response(status_code=400, body="Invalid data level")
-
-    # Add data to the file catalog and status tables
-    # create status params
-    # Event time looks like this:
-    # "time": "2024-01-16T17:35:08Z"
-    # Parses the time string from the event to a datetime object.
-    ingestion_date = datetime.datetime.strptime(event["time"], "%Y-%m-%dT%H:%M:%SZ")
-    # Formats the datetime object to a string with the format "%Y%m%d".
-    ingestion_data_str = ingestion_date.strftime("%Y%m%d")
-    # Parses the newly formatted string back into a datetime
-    # object with the desired structure.
-    ingestion_date_object = datetime.datetime.strptime(ingestion_data_str, "%Y%m%d")
-    status_params = {
-        "file_path_to_create": s3_filepath,
-        "status": models.Status.SUCCEEDED,
-        "job_definition": None,
-        "ingestion_date": ingestion_date_object,
-    }
-    try:
-        logger.info(f"Inserting {filename} into database")
-        update_status_table(status_params)
-        logger.info("Wrote data to status table")
-    except Exception as e:
-        logger.error(str(e))
-        return http_response(status_code=400, body=str(e))
-
-    # Query to get foreign key id for catalog table
-    status_tracking = None
-    with Session(db.get_engine()) as session:
-        # Query to get foreign key id for catalog table
-        query = select(models.StatusTracking.__table__).where(
-            models.StatusTracking.file_path_to_create == s3_filepath
-        )
+    ingestion_date_object = get_file_creation_date(s3_filepath)
 
-        status_tracking = session.execute(query).first()
+    metadata_params["ingestion_date"] = ingestion_date_object
+    update_file_catalog_table(metadata_params)
+    logger.info("Wrote data to file catalog table")
```

Comment:
`info()` calls should probably contain some more information about what is going on, like the filepath, ingestion date you created or something like that if you wanted to know that. This seems more like a debug level that we don't really care about in normal logs, but may want to see how it progresses if we increase the logging level.
```suggestion
    logger.debug("Wrote data to file catalog table")
```